.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+#### Pre-Flight checklist
+
+- [ ] Screenshots and design review for any changes that affect layout or styling
+  - [ ] Desktop screenshots
+  - [ ] Mobile width screenshots
+- [ ] Migrations
+  - [ ] Migration is backwards-compatible with current production code
```

Comment:
I don't think this is relevant to this repo?

Reply:
I agree we won't be having migrations in this repo generally.